README.md

```diff
@@ -1,15 +1,28 @@
-# nimbus-theme
-The best dark theme for Emacs
+# Nimbus theme for Emacs
 
-## Demos
+The best dark theme for Emacs.
 
-[Python](http://i.imgur.com/2Ij1ROs.png)
+## Screeshots
 
-[Elisp](http://i.imgur.com/iSXPuqB.png)
+Python:
 
-[Nim](https://i.imgur.com/nPLOtwL.png)
+![Python](http://i.imgur.com/2Ij1ROs.png)
```

Comment:
I'm not sure about inlining the images, as they take up a lot of space and can be a pain to scroll past. Adding a table of contents might help, but the images would still autoload and be annoying for people. I would suggest inlining one image maximum, and keeping the rest as links. Thanks for the contribution!